### DISCUSSION TITLE
Is is safe to use fluentUI classes for styling?

### DISCUSSION THREAD

**Discussion by ishantsolanki-vakt** [microsoft/fluentui#21499] [Q&A] [answered]
Given a style declared like this
```
export const styleList = mergeStyles({
  root: {
    '.ms-DetailsHeader': {
      background: '#252321',
      paddingTop: 0,
      borderBottomColor: '#F3F2F1',
      fontSize: '14px',
    },
    '.ms-DetailsRow-cell': {
      background: '#252321',
      color: '#F3F2F1',
      fontSize: '12px',
      borderBottom: '1px solid #65615D',
      height: '32px',
    },
    '::-webkit-scrollbar': {
      display: 'none',
    },
  },
});
```

Is it safe to override styles using the predefined classes generated by FluentUI?
For e.g. overriding styles on `ms-DetailsHeader`, `ms-DetailsRow-cell` etc.
I am concerned that this might break when we upgrade to a newer version of fluentUI as these classes might be renamed/omitted.

**Comment by khmakoto** [comment, marked as answer]
I can confidently say that we won't rename these classes in a patch or minor version bump. This would be considered a breaking change and we would only do something like this in a major version bump.
